lab1/README.md

```diff
@@ -100,7 +108,7 @@ $ jq .method lab1/experiment.json
     "provider": {
       "type": "process",
       "path": "vegeta",
-      "arguments": "attack -targets=experiments/data/middle_service_restarts.txt -workers=1 -rate=2 -timeout=3s -duration=40s -output=experiments/data/middle_service_restarts.bin"
+      "arguments": "attack -targets=lab1/vegeta.txt -workers=1 -rate=3 -timeout=2s -duration=40s -output=lab1/vegeta_results.bin"
```

Comment:
interesting, why the change of rates/timeouts?

Reply:
This is the actual section taken from the experiment.json. It seems the files have changed since this was updated